### DISCUSSION TITLE
Missing TOP and SKIP methods in SqlBuilder

### DISCUSSION THREAD

**Discussion by azachert** [maxtoroq/DbExtensions#49] [Q&A]
Hi,

Are you planning to add TOP and SKIP methods to SQL Builder?

BR,

**Comment by maxtoroq** [comment]
A method for TOP doesn't make much sense, since it goes between the SELECT clause name and the column list, e.g.:

``` csharp
var query = SQL
   .SELECT("TOP {0} *", pageSize)
   ...
```

About SKIP, perhaps you are confusing with LINQ, if that's the case consider using SqlSet instead.
